app/components/collections/edit/form_component.html.erb

```diff
@@ -0,0 +1 @@
+<%= render partial: 'collections/form_fields', locals: { collection_form: @collection_form } %>
```

Comment:
This is matching the pattern for works where this was required for turbo streams.

Reply:
To the best I can tell, there's no reason for this to be a component.